memory_app_fastflash.ld

```diff
@@ -0,0 +1,9 @@
+/* STM32F205RE - 512K Flash, 128K RAM */
+/* program starts at 0x2001200 */
+MEMORY
+{
+	rom (rx) : ORIGIN = 0x20012000, LENGTH = 40K
```

Comment:
I originally started at 64K, dropped it down to 48K, and instead of dropping down to 40K, I accidentally brought it up to 56K!